database/seeds/ReviewSeeder.php

```diff
@@ -13,16 +14,13 @@ class ReviewSeeder extends Seeder
      */
     public function run()
     {
-        $reviews = factory(\Hedonist\Entities\Review\Review::class, 10)->make();
-        $reviews->map(function ($item) {
-            $item->user_id = User::all()->random()->getKey();
-            $item->place_id = Place::all()->random()->getKey();
-            $item->save();
+        $places = Place::all();
+
+        $places->map(function ($place) {
+            factory(Review::class)->create([
+                'user_id' => User::all()->random()->getKey(),
```

Comment:
Count that if migration of users hadn't primary key setup, this wouldn't work `getKey()`, random()->id is more readable.